os/drivers/audio/alc5658.c

```diff
@@ -949,23 +959,14 @@ static int alc5658_ioctl(FAR struct audio_lowerhalf_s *dev, int cmd, unsigned lo
 	/* Deal with ioctls passed from the upper-half driver */
 
 	switch (cmd) {
-	/* Check for AUDIOIOC_HWRESET ioctl.  This ioctl is passed straight
-	 * through from the upper-half audio driver.
-	 */
 
 	case AUDIOIOC_HWRESET: {
-		/* REVISIT:  Should we completely re-initialize the chip?   We
-		 * can't just issue a software reset; that would puts all ALC5658
-		 * registers back in their default state.
-		 */
-
-		audvdbg("AUDIOIOC_HWRESET:\n");
+		/* This should put ALC5658 in default state, reconfiguration needed */
+		audvdbg("AUDIOIOC_HWRESET: No Action Would be taken now \n");
```

Comment:
We have a alc5658_hw_reset(), but we don't have to use here?

Reply:
I think it is better to keep it.
Currently unused, it can be used according to the driver.